src/utilities/phone/index.test.ts

```diff
@@ -107,3 +108,25 @@ describe('isValid', () => {
     });
   });
 });
+
+describe('format', () => {
+  test('should format phone', () => {
```

Comment:
I think it would be interesting to add tests using a string with non numeric characters and strings with length larger than a phone number length.
i.e.:
`expect(format('11988#887AB777')).toBe('(11) 9888-877777');
expect(format('11988887777123123123')).toBe('(11) 9888-877777')`

Reply:
after the change suggested by @hyanmandian this test case:
> `expect(format('11988887777123123123')).toBe('(11) 9888-877777')`

will be impossible, instead it'll return:

```+11 (98) 88877-77123123123```

is this ok?